snap/snapcraft.yaml

```diff
@@ -1,22 +1,20 @@
-name: tldr # you probably want to 'snapcraft register <name>'
-base: core22 # the base snap is the execution environment for this snap
-version: '3.2.0' # just for humans, typically '1.2+git' or '1.3.2'
-summary: tldr python client # 79 char long summary
+name: tldr
+base: core24
+version: '3.2.0'
```

Comment:
```suggestion
version: '3.3.0'
```

Reply:
I am sorry to spam the thread because  I forgot a lot of good habits to the proper review process. I am re-learning it.